### ISSUE TITLE
Proposed changes to the Cosmos Hub/Validators Sections 

### ISSUE THREAD

**Issue by Catdotfish** [cosmos/gaia#1154]
## Summary

<Proposed changes to the Cosmos Hub/Validators Sections >

## Problem Definition

<Every day we get requests for the documentation needed to set up a Cosmos Hub validator for the first time, the specifications needed (hardware, Software,...), and the workload associated with it. Unfortunately, the documentation used so far and available on the official Cosmos Hub site contains errors, is not always easily comprehensible or outdated. Modifications are therefore proposed so that the documentation allows those who approach cosmos as potential validators the smoothest experience possible, reflecting the idea of inclusiveness that has always distinguished the Cosmos Network.>

## Proposal

<**Validators Overview**

#[introduction](https://hub.cosmos.network/main/validators/overview.html#introduction)
_Validator candidates can bond their own ATOM and have ATOM "delegated", or staked, to them by token holders. The Cosmos Hub has 125 validators, but over time the number of validators will increase to 300 according to a predefined schedule. The validators are determined by who has the most stake delegated to them — the top 125 validator candidates with the most stake are the current Cosmos validators._
• The validators’ active set on Cosmos Hub is formed by the top 150 validators per delegated tokens, not 100 nor 125 as I see written multiple times in the docs. 
• Makes little sense to keep mentioning a schedule that will bring the validators’ active set to 300: looking to the whitepaper roadmap this was supposed to happen a long time ago, and realistically now we have no idea even if it's going to happen or when. For sure non in the upcoming quarters. At the same time, seems something certain, when in reality it’s all about Cosmos Hub Governance confirming their willingness to increase the number of the validators or not. So seems to be better or to remove that mention or to stay vaguer with a _validators’active set might be implemented over time through governance proposal_

_Initially, transaction fees are paid in ATOM but in the future, any token in the Cosmos ecosystem will be valid as fee tender if the token is whitelisted by governance._ 
• I would rephrase this as it doesn't reflect the reality in my opinion: we can say that more tokens might be whitelisted through governance, but not any token. Plus, would open up again to the same old questions as “so why I’m holding ATOMs if even Cosmos Hub transactions are paid with other tokens?”
 
_If validators double sign, are frequently offline or do not participate in governance, their staked ATOM (including ATOM of users that delegated to them) can be slashed. The penalty depends on the severity of the violation._
• There is no penalty for validators who are not active on Governance, so the above statement is untrue.

[#hardware](https://hub.cosmos.network/main/validators/overview.html#hardware)
_Initial network requirements can be low. The current testnet requires minimal resources. Then bandwidth, CPU, and memory requirements rise as the network grows. Large hard drives are recommended for storing years of blockchain history._
• We are not anymore in the testnet phase, so would be good to point people out to the active testnet docs.
• When we talk about the requirements are way too vague: what I should understand from "large hard drives are recommended". Large is a subjective adjective, we need to give people precise suggestions so that they run the validator smoothly.

[#set-up-a-website](https://hub.cosmos.network/main/validators/overview.html#set-up-a-website)
[#seek-legal-advice](https://hub.cosmos.network/main/validators/overview.html#seek-legal-advice)
_Set up a dedicated validator's website and signal your intention to become a validator on the Cosmos Forum Validator Candidates Site_ & _Seek legal advice if you intend to run a Validator._
• Setting up a website shouldn't be perceived as mandatory but as suggested practice, as well as looking for legal advice.
• Delete the link to the old validators thread on the forum, which isn’t updated anymore and people look on explorers’ description as the first approach, landing directly to the validators’ socials.

[#community](https://hub.cosmos.network/main/validators/overview.html#community)
• Replace validators’ Riot chat & forum with Cosmos Discord Server invite link and process to join the validator verified channel. Maybe it’s worthy to have a link to request to be added to the validators’ mailing list?

**Run a Validator on the Cosmos Hub Mainnet**

_Information on how to join the mainnet (genesis.json file and seeds) is held in our launch repo._
• Outdated. Isn’t better to update this page in the contents rather than confuse people?[ Or just redirect to what Billy wrote here for the mainnet upgrade?](https://github.com/cosmos/mainnet)
• I would add a link to the[ snapshot page](https://cosmos.quicksync.io/).
• I would mention the difference between running a full node and being a validator as I received this question multiple times
• Would mention a note about on-chain upgrades that we are now performing, which differ much from the previous ones
I would mention somewhere that parameters as slashing conditions, active set, etc are subjective of each blockchain, underlying the ones valid for the cosmos hub.

[#what-is-a-validator](https://hub.cosmos.network/main/validators/validator-setup.html#what-is-a-validator)
_Validators are responsible for committing new blocks to the blockchain through voting._ 
• If I read this seems that to commit a new block you need to vote in governance, while a validator can be a proposer and validate/sign a block. 
 
[#create-your-validator](https://hub.cosmos.network/main/validators/validator-setup.html#create-your-validator)
• We should write more clearly that some parameters can not be changed once having set up the validator

_You can confirm that you are in the validator set by using a third party explorer_
• We should add the links to Mintscan and BigDipper

[#participate-in-genesis-as-a-validator](https://hub.cosmos.network/main/validators/validator-setup.html#participate-in-genesis-as-a-validator)
• There is little sense in keep having this section
 
[#confirm-your-validator-is-running](https://hub.cosmos.network/main/validators/validator-setup.html#confirm-your-validator-is-running)
• Mention the alert bot for Validators we have on discord 

_To be in the validator set, you need to have more total voting power than the 100th validator._
• Taking that the active validators ser is formed by 150 validators, I rephrase in something like [As validator you need to have in total more ATOM staked/bonded/delegated (from you or other delegators) than the last validator actually listed in the active set. I would also add a link to Mintscan or BigDipper Validators’ page.

**Validator FAQ**

[#what-is-staking](https://hub.cosmos.network/main/validators/validator-faq.html#what-is-staking)
_Initially, only the top 125 validators with the most voting power are active validators._
• Only the top 150 validators…
 
[#what-is-a-full-node](https://hub.cosmos.network/main/validators/validator-faq.html#what-is-a-full-node)
• Can we add what are the specifications about hardware etc that are different for a full node to run smoothly compared to a light node and each one use-case?
 
[#becoming-a-validator](https://hub.cosmos.network/main/validators/validator-faq.html#becoming-a-validator)
• Would be good to remind again at the very beginning that they have to be careful as some parameters can not be changed once set up.
 
_After a validator is created, ATOM holders can delegate ATOM to them, effectively adding stake to the validator's pool._
• In fact, delegators, other validators, and the validator itself fac delegate to the pool one created, but I would like to highlight the fact that will be possible only via CLI as Cosmos Wallets only shows the Validators which are already part of the active set.
 
_The 125 validators with the most total stake are the designated validators. If a validator's total stake falls below the top 125, then that validator loses their validator privileges. The validator cannot participate in consensus or generate rewards until the stake is high enough to be in the top 125._
• 150 validators…
• typo: their instead that its

[#how-can-i-join-the-testnet](https://hub.cosmos.network/main/validators/validator-faq.html#how-can-i-join-the-testnet)
_The testnet is a great environment to test your validator setup before launch. Testnet participation is a great way to signal to the community that you are ready and able to operate a validator. For details, see Join the Public Testnet documentation and the https://github.com/cosmos/testnets_
• can’t we just link to the updated GitHub repo (if any?) it might be confusing to be pointed out in two different directions in which the instruction might differ on GitHub.
 
[#what-are-the-different-states-a-validator-can-be-in](https://hub.cosmos.network/main/validators/validator-faq.html#what-are-the-different-states-a-validator-can-be-in)
_If the jailing is due to being offline for too long, the validator can send an unjail transaction in order to re-enter the validator set._
• Can’t we add how many blocks the validator has to miss? It’s a bit generic as it is now.
 
[#how-do-delegators-choose-their-validators](https://hub.cosmos.network/main/validators/validator-faq.html#how-do-delegators-choose-their-validators)
_A high voting power shows that the community trusts this validator, but it also means that this validator is a bigger target for hackers._
• We seem to suggest that somehow validators can be hacked and funds can be stolen…? Or we are more specific or I don’t think makes much sense having this.
 
_Commission rate: Commission applied on revenue by validators before the revenue is distributed to their delegators._
• Would be possible to add a practical example?
 
_Track record: Delegators review the track record of the validators they plan to delegate to. This track record includes seniority, past votes on proposals, historical average uptime, and how often the node was compromised._
• Afaik there is no way to have a track of the majority of these points, so it makes no sense to mention it.
 
_For example, a good practice for validators is to have a third party audit their setup. Note though, that the Tendermint team does not approve or conduct any audit themselves. For more information on due diligence, see the A Delegator’s Guide to Staking blog post._
• I don’t think that people ask themselves if their validator ever did an audit, and the fact that Tendermint doesn’t conduct audits makes no sense to be here. 
• Doesn’t make much sense a delegators outdated guide in the validators section

I would add these points to the things that might increase the validators’ value into delegators’ eyes:
• Create educational content: such as tutorials, videos, guides, blog posts, AMAs, podcasts, etc. 
• Stay active in the community channels and willing to help troubleshoot the delegators’ issues
• Develop/ Maintain Cosmos Specific solutions as Wallets, Explorer, Relayers system, …
• Stay active in the Governance Process and keep the delegators updated through socials.
• Slash Refund Policy
•  It’s a good practice to avoid delegating to validators that represent centralized exchanges as they do not participate in Governance nor the development of the Network.
• Professional 24/7 environment able to handle any potential issue promptly.

[#can-a-validator-run-away-with-their-delegators-atom](https://hub.cosmos.network/main/validators/validator-faq.html#can-a-validator-run-away-with-their-delegators-atom)
_Even though delegated funds cannot be stolen by their validators, delegators are still liable if their validators misbehave._
• As in the first part we are talking about the whole delegator's funds, this phrase sounds like the whole stake is at risk, while there is a loss of just a small portion in the circumstance that the validator gets slashed.
 
[#are-validators-of-the-cosmos-hub-required-to-validate-other-zones-in-the-cosmos-ecosystem](https://hub.cosmos.network/main/validators/validator-faq.html#are-validators-of-the-cosmos-hub-required-to-validate-other-zones-in-the-cosmos-ecosystem)
_Yes, governance requires validators to validate additional zones in the Cosmos ecosystem_
• This is not true at all, there is no such requirement, it's totally optional and personal decide if and which other cosmos chains a validator wants to validate.
 
[#what-is-the-incentive-to-run-a-validator](https://hub.cosmos.network/main/validators/validator-faq.html#what-is-the-incentive-to-run-a-validator)
• This part seems to me not very incentivizing as transaction fees and block rewards are already a little thing for smaller validators, so if we say that they have to even split them with validators seems really not a possible business. And even Governance seems more a duty than something valuable put in these terms. Maybe write the paragraph with a more Interchain perspective assuming the growth of the ecosystem, Interchain Security, and so on?
 
[#what-are-the-slashing-conditions](https://hub.cosmos.network/main/validators/validator-faq.html#what-are-the-slashing-conditions)
• Would be possible to write an approx time window regarding the 10k blocks that have to be missed before having a 0.1% slash?

[ #how-to-prevent-concentration-of-stake-in-the-hands-of-a-few-top-validators](https://hub.cosmos.network/main/validators/validator-faq.html#how-to-prevent-concentration-of-stake-in-the-hands-of-a-few-top-validators)
_UI warning: Wallets can implement warnings that are displayed to users when they want to delegate to a validator that already has a significant amount of staking power._
• This is not a feature present in the Cosmos Wallets and I don’t think this is in the plan of the Cosmostation and Keplr teams in the short period, so I would just delete it.
• Instead, we can mention airdrop distribution criteria as excluding the top 20, awareness work from the community managers, and so on.

[#technical-requirements](https://hub.cosmos.network/main/validators/validator-faq.html#technical-requirements)
• This part should be improved with specific requirements that will guarantee to validators to run their business smoothly. Example:
_A modest level of hardware specifications is initially required and rises as network use increases. Participating in the testnet is the best way to learn more._
It tells nothing about what I, a candidate validator, should prepare to get started on my journey.

[#how-to-handle-key-management](https://hub.cosmos.network/main/validators/validator-faq.html#how-to-handle-key-management)
• Is Ledger X an option as well?
• We should remove all the _The Tendermint team does not recommend one solution above the other_  that we can spot in these pages or substitute _Tendermint Team_ with _Interchain Foundation_
 
[#what-are-the-maintenance-requirements](https://hub.cosmos.network/main/validators/validator-faq.html#what-are-the-maintenance-requirements)
• The best way to stay updated about this is to join the Cosmos Hub Validators Verified channels on Discord, which will be directly monitored by the Cosmos Hub Team and its Lead.
>

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by nooomski** [comment]
Solved most of these comments in #1207 . Closing this issue, we can continue discussion in that PR. Thanks @Catdotfish !
